prep-javascript-conditionals/main.js

```diff
@@ -0,0 +1,74 @@
+function isAdult(age) {
+  if (age > 18) {
```

Comment:
What about if someone is 18? That would also make them an adult

Reply:
updated this in recent commit!